tests/Dockerfile_unittest

```diff
@@ -21,23 +22,26 @@ RUN apk update \
       z3 \
       # use deploy.sh
       jq \
-      expect
+      expect \
+      # nodejs
+      nodejs \
+      npm
```

Comment:
I think there are various ways to install it, but I decided to install it using the simplest way. The downside is that we can't specify a specific version, but I don't think it's a big problem since it's a testing container.